client/my-sites/theme/main.jsx

```diff
@@ -37,24 +39,42 @@ const ThemeSheet = React.createClass( {
 		descriptionLong: React.PropTypes.string,
 		supportDocumentation: React.PropTypes.string,
 		taxonomies: React.PropTypes.object,
+		isLoggedIn: React.PropTypes.bool,
+		// Connected props
+		selectedSite: React.PropTypes.object,
+		siteSlug: React.PropTypes.string,
 	},
 
 	getDefaultProps() {
-		return { section: 'details' };
+		return { section: 'overview' };
 	},
 
 	onBackClick() {
 		page.back();
 	},
 
+	isPremium() {
+		return this.props.price.value > 0;
+	},
```

Comment:
It's kinda unfortunate that we're duplicating the `isPremium` predicate from `my-sites/themes/helpers` again here (albeit with a different implementation tailored to the details endpoint). Eventually, this should become a selector -- wondering whether it might make sense to move it to `state/themes/theme-details/selectors` already?